tests/behaviour/steps/params.rs

```diff
@@ -298,14 +303,38 @@ impl FromStr for ObjectRootLabel {
     type Err = String;
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         let kind = match s {
-            "entity" => TypeDBTypeKind::Entity,
-            "relation" => TypeDBTypeKind::Relation,
+            "entity" | "entities" => TypeDBTypeKind::Entity,
+            "relation" | "relations" => TypeDBTypeKind::Relation,
             _ => unreachable!(),
         };
         Ok(Self { kind })
     }
 }
 
+#[derive(Debug, Parameter)]
+#[param(name = "root_label_extended", regex = r"(attribute|entity|relation|role|object)")]
+pub(crate) enum RootLabelExtended {
```

Comment:
Can we rename this to `Kind` instead of `RootLabel`? Also probably needs to be done in BDD and implementations thereof... to keep this PR from growing more, maybe you can make a note to yourself to make a little follow up one to replace Root with Kind in BDD/place you find it in Core

Reply:
Will look at it

Reply:
Created a task